app/src/main/res/layout/activity_itinerary_create.xml

```diff
@@ -27,8 +32,73 @@
             android:layout_margin="30sp"
             android:clickable="true"
             android:contentDescription="@string/content_descriptor_save_icon"
-            android:onClick="goToSaveActivity"
+            android:onClick="onSaved"
             app:srcCompat="@android:drawable/ic_menu_save" />
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/showMission"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|end"
+            android:layout_marginEnd="30sp"
```

Comment:
Using androidx.constraintlayout.widget.Guideline and then add constraints between them rather than hardcoding sp/px values would be cleaner and might give better results. You don't absolutely need to change it now, as many UI things will probably need to be changed at the end of the semester to make the app have an overall style, but if you write a new XML layout for an activity later on, it might be a better practice

Reply:
Yes, you are right, this should be a task for a next sprint